tests/PHPUnit/Framework/Fixture.php

```diff
@@ -77,7 +77,7 @@
 class Fixture extends \PHPUnit_Framework_Assert
 {
     const IMAGES_GENERATED_ONLY_FOR_OS = 'linux';
-    const IMAGES_GENERATED_FOR_PHP = '5.6';
+    const IMAGES_GENERATED_FOR_PHP = '7.2';
```

Comment:
Because of that some scheduled report tests were not executed at all. Adjusted the expected files as well, so the tests pass again :-)